build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val root = (project in file("."))
   .settings(
     organization := "org.broadinstitute.dsp",
     name := "helm-scala-sdk",
-    version := "0.0.1-SNAPSHOT",
+    version := "0.0.2-SNAPSHOT",
```

Comment:
Should we have a CHANGELOG? 🤔

Reply:
I'm thinking maybe we should just use release instead of changelog

Reply:
also since this is `SNAPSHOT`, I think we don't have to bump version...

Reply:
Good point, maybe will bump back to `0.0.1-SNAPSHOT` then.

I think I bumped to 0.0.2 partly because I was having trouble clearing my maven caches 🤦

Reply:
maybe we can do `0.0.1-RC1` 🤔